strelka/strelka.cwl

```diff
@@ -2,35 +2,43 @@
 
 cwlVersion: v1.0
 class: CommandLineTool
-label: "strelka 1.0.15"
-baseCommand: "/usr/bin/cwl_helper.sh"
+label: "strelka 2.7.1"
+baseCommand: "/usr/bin/docker_helper.pl"
 requirements:
     DockerRequirement:
-        dockerPull: "mgibio/strelka-cwl:1.0.15"
+        dockerImageId: "mgibio/strelka-cwl:2.7.1"
```

Comment:
I changed as part of testing with a locally tagged version (see also the `cwltool` `--disable-pull` option).  I should put it back 😄